proxy/coco/tests/replication.rs

```diff
@@ -193,43 +220,46 @@ async fn can_fetch_project_changes() -> Result<(), Box<dyn std::error::Error>> {
         };
 
         {
-            let results = alice_state.transport_results();
-            let mut rad = repo.find_remote(config::RAD_REMOTE)?;
-            rad.push(&[&format!("refs/heads/{}", project.default_branch())], None)?;
-            assert!(results.wait(Duration::from_secs(3)).is_some());
+            let mut rad =
+                Remote::<LocalUrl>::rad_remote::<_, Fetchspec>(LocalUrl::from(project.urn()), None);
+            let _ = rad.push(
+                state::settings(&alice_peer),
+                &repo,
+                LocalPushspec::Matching {
+                    pattern: RefLike::try_from(format!(
+                        "refs/heads/{}",
+                        project.subject().default_branch.clone().unwrap()
+                    ))
+                    .unwrap()
+                    .into(),
+                    force: Force::False,
+                },
```

Comment:
I’m not a hundred percent certain but using `LocalPushspec::Configured` would work too and would be a lot easier to read.

Reply:
It's been a while _but_ I _think_ I tried that. I could try again and see

Reply:
Didn't work, but I could make it look a bit nicer at least